rules/awesome-readme/rule.md

```diff
@@ -91,3 +91,77 @@ A great tool to help you build your README is [GitHub Simp](https://readmi.xyz/)
 :::
 
 You can also try [readme.so](https://readme.so/), which offers a similar service with more customization options which is maintained by [Katherine Oelsner](https://github.com/octokatherine) from GitHub.
+
+## More examples
+
+::: greybox
+
+#### GordonBeemingCom
+
+- Title
+- Description
+- _How to add database migration_
+
+![](bad-example-gordonbeemingcom.jpg)
```

Comment:
Hi @GordonBeeming 

1. Please add the caption here ` ![Caption](imagefile.jpg)`
2. Move the link to be right below the heading, above the list
3. Do the same for other examples

Reply:
I left the caption off because then it would duplicate figures because the grey box has a good or bad figure
Links don't work inside the figure so it would look misplaced if just floating